tox.ini

```diff
@@ -52,8 +70,12 @@ passenv = LANG TERM LANGUAGE LC_ALL LD_LIBRARY_PATH BRANCH
 deps =
 	pip-tools
 	coverage
+	-r requirements/tests.txt
 commands =
 	python -c "import fcntl; fcntl.fcntl(1, fcntl.F_SETFL, 0)"
-	pip-sync requirements/base.txt
 	pip install -e .
+	pip uninstall -y d3a-interface
+	pip install git+https://github.com/gridsingularity/d3a-interface.git@{env:BRANCH:master}
```

Comment:
Why do we need to uninstall/reinstall d3a-interface, if it's one of the packages installed with `pip install -e .`?

Reply:
THis is true here, however in other repos where we synced the d3a-interface requirements in the requirements.txt files, the d3a-interface master branch was the default and was installed and not being overriding in the pip install -e .
So it's safe here for now